pkg/controller/collection/collection_controller.go

```diff
@@ -652,6 +690,8 @@ func activatev2(collectionResource *kabanerov1alpha1.Collection, collection *Ind
 			transformedManifests = append(transformedManifests, transformedRemoteManifest{m, asset.Url})
 		}
 
+// 182: Do not delete, only apply
+/*
```

Comment:
I am not a big fan of commenting out code that's under version control.  If we've decided that we're not going to do it, I would prefer to just delete it, because someone can come back later and look at the prior version in Git to see what it used to look like.  Again, feel free to ignore this comment if others disagree.

Reply:
Ok, will clean that up

Reply:
removed commented code in  commit https://github.com/kabanero-io/kabanero-operator/pull/186/commits/d71fb952c78211209a2063be652e0ab60d62e2ac